app/views/periodSummaryPastReturns.scala.html

```diff
@@ -82,6 +83,28 @@
           </div>
 
         } @*liability-returns end*@
+
+          <div class="form-group">
+          @ReliefsUtils.partitionNewestReliefForType(b.reliefReturns)._2.zipWithIndex.map { t =>
```

Comment:
This logic should be done at the service level.

I'm thinking PeriodSummaryController.view should be asking summaryReturnsService to call the util you've put in and all the view needs to do is pull out what it needs.